services/src/main/resources/db/changelog/gamification.db.changelog-1.0.0.xml

```diff
@@ -0,0 +1,156 @@
+            <column name="CREATED_DATE" type="timestamp" defaultValueDate="${now}">
+                <constraints nullable="false"/>
+            </column>
+            <column name="LAST_MODIFIED_BY" type="varchar(200)"/>
+            <column name="LAST_MODIFIED_DATE" type="timestamp"/>
+        </createTable>
+        <modifySql dbms="mysql">
+            <append value=" ENGINE=INNODB CHARSET=UTF8 COLLATE utf8_general_ci"/>
+        </modifySql>
+    </changeSet>
+    <changeSet author="task" id="1.0.0-2" dbms="oracle,postgresql">
+        <createSequence sequenceName="SEQ_GAMIFICATION_BADGE_ID" startValue="1"/>
+    </changeSet>
+    <changeSet author="exo-gamification" id="1.0.0-2">
+        <createTable tableName="GAMIFICATION_RULE">
+            <column name="ID" type="bigint" autoIncrement="${autoIncrement}">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="TITLE" type="NVARCHAR(70)">
+                <constraints nullable="false" unique="true"/>
+            </column>
+            <column name="DESCRIPTION" type="NVARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="SCORE" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="AREA" type="NVARCHAR(32)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="ENABLED" type="BIT" defaultValue="0">
+                <constraints nullable="false"/>
+            </column>
+            <column name="CREATED_BY" type="varchar(200)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="CREATED_DATE" type="timestamp" defaultValueDate="${now}">
+                <constraints nullable="false"/>
+            </column>
+            <column name="LAST_MODIFIED_BY" type="varchar(200)"/>
+            <column name="LAST_MODIFIED_DATE" type="timestamp"/>
+        </createTable>
+        <modifySql dbms="mysql">
+            <append value=" ENGINE=INNODB CHARSET=UTF8 COLLATE utf8_general_ci"/>
+        </modifySql>
+    </changeSet>
+    <changeSet author="exo-gamification" id="1.0.0-3" runOnChange="true">
+        <createTable tableName="GAMIFICATION_USER_REPUTATION">
+            <column autoIncrement="${autoIncrement}" name="ID" type="BIGINT">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="SCORE" type="BIGINT"/>
+            <column name="USERNAME" type="VARCHAR(200)">
+                <constraints nullable="false" unique="true"/>
+            </column>
+        </createTable>
+        <modifySql dbms="mysql">
+            <append value=" ENGINE=INNODB CHARSET=UTF8 COLLATE utf8_general_ci"/>
+        </modifySql>
+    </changeSet>
+    <changeSet author="exo-gamification" id="1.0.0-4" runOnChange="true">
+        <createTable tableName="GAMIFICATION_CONTEXT_ITEM">
```

Comment:
Use plural for table names, see:
https://community.exoplatform.com/portal/intranet/wiki/group/spaces/engineering/RDBMS_Guidelines

Reply:
Done